src/List.h

```diff
@@ -264,8 +259,8 @@ class List {
 
 	void push_back(const T& a)	{ append(a); }
 	void push_front(const T& a)	{ insert(a); }
-	T pop_front() { return get() ;}
-	T pop_back() { return remove_nth(num_entries-1); }
+	void pop_front()	{ remove_nth(0); }
+	void pop_back()	{ remove_nth(entries-1); }
```

Comment:
`pop_back` now looking wrong: should be `remove_nth(num_entries-1)` ?

Reply:
🤦‍♂